hack/dockerfile/install/containerd.installer

```diff
@@ -4,7 +4,7 @@
 # containerd is also pinned in vendor.conf. When updating the binary
 # version you may also need to update the vendor version to pick up bug
 # fixes or new APIs.
-CONTAINERD_COMMIT=9f2e07b1fc1342d1c48fe4d7bbb94cb6d1bf278b # v1.1.4
+CONTAINERD_COMMIT=aa537a67b3a9026c423a63dbaf71e4cc0776b2ad # v1.2.0-13-g8afcade1
```

Comment:
may want to remove the `# v1.2.0-13-g8afcade1` comment, because it no longer matches 😅

Reply:
Mainly because the version is pinned on a commit (related to https://github.com/moby/moby/pull/37968), which makes it difficult to see what version of containerd is being used.